createdisk.sh

```diff
@@ -320,6 +320,15 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
     echo hv_sock > /etc/modules-load.d/vsock.conf
 EOF
 
+# Add gvisor-tap-vsock service
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} 'sudo bash -x -s' <<EOF
+  podman run -d --name=gvisor-tap-vsock --privileged --net=host -it quay.io/crcont/gvisor-tap-vsock:v1
+  podman generate systemd --restart-policy=no gvisor-tap-vsock > /etc/systemd/system/gvisor-tap-vsock.service
```

Comment:
@guillaumerose why `restart-policy` is set to `no`, I think in case of container crashed then shouldn't it need restart?

Reply:
I don't want to pollute too much regular installation of SNC. So, if after 100 tries (with a sleep of 1s), it still can't connect, the container die. But it will still retry at the next startup.

Reply:
I changed the behaviour: now if some particular network interfaces are present, the container will shut down.

Reply:
if the process/pod crashes?

Reply:
yes if it crashes, it's over. No way to contact the VM. 
I would expect that it doesn't happen and code doesn't panic and properly returns an error and fallback in the retry loop.